packages/MarkdownEditor-Core.package/MarkdownCommentBlock.class/instance/commentEnd.st

```diff
@@ -0,0 +1,8 @@
+accessing
+commentEnd
+	
+	| closingTag |
+	closingTag := self closingTag.
```

Comment:
As the message is short enough, the variable could be removed in favour of writing `self closingTag`.